2023-05-11T01:38:38.6250482Z ##[group]Run actions/setup-python@v4
2023-05-11T01:38:38.6250856Z with:
2023-05-11T01:38:38.6251180Z   python-version: 3.8
2023-05-11T01:38:38.6251452Z   architecture: x64
2023-05-11T01:38:38.6251798Z   cache: pip
2023-05-11T01:38:38.6252107Z   check-latest: false
2023-05-11T01:38:38.6252583Z   token: ***
2023-05-11T01:38:38.6252852Z   update-environment: true
2023-05-11T01:38:38.6253204Z   allow-prereleases: false
2023-05-11T01:38:38.6253560Z ##[endgroup]
2023-05-11T01:38:38.8121067Z ##[group]Installed versions
2023-05-11T01:38:38.8223881Z Successfully set up CPython (3.8.16)
2023-05-11T01:38:38.8224879Z ##[endgroup]
2023-05-11T01:38:39.5972253Z [command]/opt/hostedtoolcache/Python/3.8.16/x64/bin/pip cache dir
2023-05-11T01:38:40.2844032Z /home/runner/.cache/pip
2023-05-11T01:38:41.3286595Z Received 6307532 of 6307532 (100.0%), 9.7 MBs/sec
2023-05-11T01:38:41.3288035Z Cache Size: ~6 MB (6307532 B)
2023-05-11T01:38:41.3323813Z [command]/usr/bin/tar -xf /home/runner/work/_temp/d0dbb602-91b2-4422-8812-4af6a34db74b/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-05-11T01:38:41.3588817Z Cache restored successfully
2023-05-11T01:38:41.3609159Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.16-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
